File: javascript/aumento.js
const salario = parseFloat(prompt("Digite o salário"))

let salarioFinal, aumento, porcentagem;

if (salario <= 1000) {
    porcentagem = 0.20;
    aumento = salario * porcentagem;
    salarioFinal = salario + aumento;
} else if (salario <= 3000) {
    porcentagem = 0.15;
    aumento = salario * porcentagem;
    salarioFinal = salario + aumento;
} else if (salario <= 8000) {
    porcentagem = 0.10;
    aumento = salario * porcentagem;
    salarioFinal = salario + aumento;
} else {
    porcentagem = 0.05;
    aumento = salario * porcentagem;
    salarioFinal = salario + aumento;
}

porcentagem *= 100;

console.log("Novo salário = R$ %.2f", salarioFinal);
console.log("Aumento = R$ %.2f", aumento);
console.log("Porcentagem = %i%%", porcentagem);

File: python/aumento.py
salario = float(input("Digite o salario da pessoa: "))
multiplicador = 0

if salario <= 1000:
    multiplicador = 0.2
elif salario <= 3000:
    multiplicador = 0.15
elif salario <= 8000:
    multiplicador = 0.1
else:
    multiplicador = 0.05

novo_salario = salario + salario * multiplicador
aumento = novo_salario - salario

print(f"Novo salario = R$ {novo_salario:.2f}")
print(f"Aumento = R$ {aumento:.2f}")
print(f"Porcentagem = {multiplicador * 100:} %")
